src/System.Net.Security/src/Resources/Strings.resx

```diff
@@ -441,4 +441,7 @@
   <data name="net_nego_server_not_supported" xml:space="preserve">
     <value>Server implementation is not supported</value>
   </data>
+  <data name="net_nego_protection_level_not_supported" xml:space="preserve">
+    <value>Requested protection level is not supported</value>
```

Comment:
We should give the user some hint as to how they can fix it, e.g.
"Requested protection level is not supported with the gssapi implementation currently installed."
or something like that.